lib/ajax-datatables-rails/datatable/simple_order.rb

```diff
@@ -20,6 +20,14 @@ def column
       def direction
         DIRECTIONS.find { |dir| dir == @options[:dir].upcase } || 'ASC'
       end
+      
+      def sort_nulls_last(sort_column)
+        if column.nulls_last == true || AjaxDatatablesRails.config.nulls_last == true
```

Comment:
`sort_column.nulls_last` ?

Reply:
the `sort_column` is just a string so that doesn't work. I need access to the column options.